drivers/overlay/overlay.go

```diff
@@ -280,7 +280,7 @@ func parseOptions(options []string) (*overlayOptions, error) {
 		trimkey = strings.TrimPrefix(trimkey, ".")
 		switch trimkey {
 		case "override_kernel_check":
-			logrus.Warnf("overlay: override_kernel_check option was specified, but is no longer necessary")
```

Comment:
I think this can be dropped and turned into a code comment? I think it has little value to users (as they don't have to worry about it).

Reply:
I'd rather not drop it entirely, but I'd be fine with it as a Debugf instead.

Reply:
Dropped it down to Debugf.